### ISSUE TITLE
bot detected 

### ISSUE THREAD

**Issue by ghost** [nmathey/finasync#23]
have a error message ...
bot detected.... 
![Capture d’écran 2024-09-10 144202](https://github.com/user-attachments/assets/fc383dae-221e-4786-9ce9-7980ff670a4e)

**Comment by nmathey** [comment]
Yeah Finary hardened its API access. As I'm using finary_uapi we are trying to see how to handle it https://[github.com/lasconic/finary_uapi/issues/187](https://github.com/lasconic/finary_uapi/issues/187)

The good news is that RealT is now on the official Finary Roadmap 
![image](https://github.com/user-attachments/assets/e01e92ec-8e3d-49c5-b7b8-110047d005b4)
